Automattic/wp-calypso
# Gutenberg: Move markdown converter into MarkdownRenderer

This PR is a small cleanup of the Markdown block that lives in Calypso that gets rid of the `MarkdownConverter` and moves the actual converter to the `MarkdownRenderer`. What the converter did previously was so simple that it just doesn't need to be in a separate util.

To test:
* Checkout this branch.
* Follow the instructions in #27098 (without the first step).
* Verify there are no functional/visual changes when compared to the version on `master`.
